### PR TITLE
Updating pulse sim from backend tutorial

### PR DESCRIPTION
**Pull request by DanPuzzuoli**
### Summary

Updating the tutorial to use the `FakeArmonk` mock backend, rather than the actual Armonk backend, to comply with tutorial requirements.

### Details and comments

The tutorial originally included first submitting the Rabi experiments job to the actual Armonk backend. This has been removed, and now just directly runs the experiments on the simulator.

### CODE REVIEW

**Comment by nonhermitian** [comment]
Many thanks for updating this.  However the docs still have a bunch of:


DeprecationWarning: Calling `GaussianSquare` with a channel is deprecated. Instantiate the new `Play` instruction directly with a pulse and a channel.

warnings in them.

**Comment by DanPuzzuoli** [comment]
Hmm odd... I'm not seeing these warnings when I run locally, even on a fresh install of the meta package (just via `pip install qiskit`; the versions are:


> Qiskit Software | Version
> -- | --
> Qiskit | 0.19.6
> Terra | 0.14.2
> Aer | 0.5.2
> Ignis | 0.3.3
> Aqua | 0.7.3
> IBM Q Provider | 0.7.2
> 
> Python | 3.8.3 (default, Jul 2 2020, 11:26:31) [Clang 10.0.0 ]
> OS | Darwin
> CPUs | 8
> Memory (Gb) | 32.0
> Wed Jul 15 09:30:42 2020 EDT
> 

I am however getting process pool failures at the simulation step, but I presume these warnings are coming from the cell calling the Ignis function `rabi_schedules`.

**Comment by nonhermitian** [comment]
If you click on the "Details" button for `Qiskit.qiskit-tutorials (Job Python38)` and then `View more details on Azure Pipelines ` and you should see "1 artifact produced". That is the generated documentation that you can download and explore.  This is how I see these warnings. Note that the tutorials are built against the latest deployed Qiskit version.

**Comment by nonhermitian** [comment]
Here is a snippet:

Screen Shot 2020-07-15 at 09 45 15

**Comment by DanPuzzuoli** [comment]
Ah wait, are these warnings the ones from the `Qiskit Aer: Pulse simulation of two qubits using a Duffing oscillator model` ? This is the only one I can find - this is actually the other pulse simulator tutorial, which also needs to be updated. Should I be moving this file out of legacy in the PR?

**Comment by nonhermitian** [comment]
Yes, legacy it just a holding place for tutorials that did not work.  Please move it to the normal tutorials repo or it will not show up.

File: legacy_tutorials/aer/9_pulse_simulator_from_backend.ipynb [hunk 1]
```diff
@@ -109,22 +113,63 @@
...
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "# set the drive strength from a previously inferred value for the real Armonk device\n",
+    "getattr(armonk_backend.configuration(), 'hamiltonian')['vars']['omegad0'] = 63455017.85\n",
+    "\n",
+    "# set the qubit frequency from the estimate in the defaults\n",
+    "freq_est = armonk_backend.defaults().qubit_freq_est[0] / 10**9\n",
```
Comment (taalexander) [inline legacy_tutorials/aer/9_pulse_simulator_from_backend.ipynb:136]: Don't hardcode this based on defaults as this is a bug. Just hardcode the frequency eg. `5e9` or whatever it is. Also 10**9 involves an exponentiation. Better to use 1e9 which is directly an integer value in Python and will be more efficient.

Comment (DanPuzzuoli) [reply legacy_tutorials/aer/9_pulse_simulator_from_backend.ipynb:136]: Thanks, will make these changes. (Especial thanks for the `1e9` tip, good to know.)

**Comment by DanPuzzuoli** [comment]
> Ohh sorry, my mistake.
> 
> Yes, legacy it just a holding place for tutorials that did not work. Please move it to the normal tutorials repo or it will not show up.

I've just moved it into the pulse tutorials folder - I'll start updating the other pulse simulator tutorial as well in a separate PR.

**Pull request merged**